mm2src/mm2_main/tests/docker_tests/docker_tests_common.rs

```diff
@@ -163,6 +163,52 @@ pub fn fill_eth(to_addr: &str) {
         .unwrap();
 }
 
+pub fn generate_eth_coin_with_random_privkey() -> EthCoin {
+    let conf = json!({"coin":"ETH","name":"ethereum","protocol":{"type":"ETH"}});
+    let req = json!({
+        "method": "enable",
+        "coin": "ETH",
+        "urls": ["http://195.201.0.6:8565"],
+        "swap_contract_address": "0xa09ad3cd7e96586ebd05a2607ee56b56fb2db8fd",
+    });
+    let priv_key = random_secp256k1_secret();
+    let priv_key_policy = PrivKeyBuildPolicy::IguanaPrivKey(priv_key);
+    block_on(eth_coin_from_conf_and_request(
+        &MM_CTX,
+        "ETH",
+        &conf,
+        &req,
+        CoinProtocol::ETH,
+        priv_key_policy,
+    ))
+    .unwrap()
+}
+
+pub fn generate_erc20_coin() -> EthCoin {
```

Comment:
How about renaming this function to `generate_jst_with_seed`, and add `seed: &str` param to it?